static/app/components/replays/replayTagsTableRow.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {ReactElement} from 'react';
+import styled from '@emotion/styled';
+import {LocationDescriptor} from 'history';
+
+import {KeyValueTableRow} from 'sentry/components/keyValueTable';
+import Link from 'sentry/components/links/link';
+import Tooltip from 'sentry/components/tooltip';
+import Version from 'sentry/components/version';
+import {EventTag} from 'sentry/types/event';
+import {AnnotatedText} from 'sentry/components/events/meta/annotatedText';
+
+type Tag = {key: string; value: string[]};
+
+interface Props {
+  generateUrl: (tag: EventTag) => LocationDescriptor;
+  query: string;
+  tag: Tag;
+  meta?: Record<any, any>;
+}
+
+function ReplayTagsTableRow({tag, query, generateUrl, meta}: Props) {
+  const keyMetaData = meta?.key?.[''];
+  const valueMetaData = meta?.value?.[''];
+
+  const renderTagValue = () => {
+    switch (tag.key) {
+      case 'release': {
```

Comment:
nit: Can we change this to an if/else? imo it's a bit easier to read esp since we only have 2 cases.